src/ngx_http_auth_jwt_module.c

```diff
@@ -613,6 +614,7 @@ static char *get_jwt(ngx_http_request_t *r, ngx_str_t jwt_location)
 {
   static const char *HEADER_PREFIX = "HEADER=";
   static const char *COOKIE_PREFIX = "COOKIE=";
+  static const char QUERY_STRING_PREFIX[] = "QUERY_STRING=";
```

Comment:
Can you update the other two here to not be pointers, as you suggested? And rename this to `QUERYSTRING_PREFIX` and update the value to be `QUERYSTRING=` rather than `QUERY_STRING=`, please.